tests/testthat/test-ppc-loo.R

```diff
@@ -73,10 +73,24 @@ test_that("ppc_loo_pit_qq returns ggplot object", {
   expect_equal(p3$labels$x, "Normal")
 })
 
-test_that("ppc_loo_pit functions work when pit specified instead of y,yrep,lw", {
+test_that("ppc_loo_pit_ecdf returns a ggplot object", {
+  skip_if_not_installed("rstanarm")
+  skip_if_not_installed("loo")
+  expect_gg(p0 <- ppc_loo_pit_ecdf(y, yrep, lw))
+  expect_gg(p1 <- ppc_loo_pit_ecdf(y, yrep, lw))
+  expect_gg(p2 <- ppc_loo_pit_ecdf(y, yrep, psis_object = psis1))
+  expect_equal(p1$labels$x, "LOO PIT")
+  expect_equal(p1$labels$y, "ECDF")
+  expect_equal(p1$data, p2$data)
+  expect_gg(p3 <- ppc_loo_pit_ecdf(y, yrep, lw, plot_diff = TRUE))
+  expect_equal(p3$labels$y, "ECDF difference")
```

Comment:
I think for future compatibility with ggplot2 we probably need to do what was done in https://github.com/stan-dev/bayesplot/pull/337/files#diff-d31e64955d39a8924be5d979cb21c5e96514987aeff9ca8cf2af067e98f3b156 with checking for the `get_labs()` function  and using it if found.

Reply:
I can also add the part for `ppc_loo_pit_qq`.